source/includes/stackpath/_workloads.md

```diff
@@ -290,45 +374,46 @@ Create a new workload in a given [environment](#administration-environments).
+`ports.publicPortSrc`<br/>*string* | A subnet that will define all the IPs allowed by the network policy rule. Defaults to `0.0.0.0/0` if not specified.
+`ports.publicPortDesc`<br/>*string* | A summary of what the network policy rule does or a name for it. It is highly recommended to give a unique description to easily identify a network policy rule. Defaults to an empty string if not provided.
+`commands`<br/>*Array[string]* | The commands that start a container. Only applicable to workloads of `type` 'CONTAINER'.
+`persistenceStoragePath`<br/>*string* | The path in an instance to mount a volume.
+`persistenceStorageSize`<br/>*int* | The size of the mounted volume (in GB).
+`addImagePullCredentialsOption` <br/>*boolean* | It is used to indicate if additional credentials to pull container image are provided or not.
+`containerUsername` <br/>*string* | The username used to authenticate the image pull.
+`containerPassword` <br/>*string* | The password used to authenticate the image pull.
+`containerServer` <br/>*string* | The server that the credentials should be used with. This value will default to the docker hub registry when not set.
+`containerEmail` <br/>*string* | The email address to use for the docker registry account
+`environmentVariables` <br/>*Array[Object]* | A list of environment variables. Only applicable to workloads of `type` 'CONTAINER'.
+`environmentVariables.key` <br/>*string* | The location to obtain a value for an environment variable.
+`environmentVariables.value` <br/>*string* | An environment variable's value.
+`secretEnvironmentVariables` <br/>*Array[Object]* | A list of sensitive environment variables. Only applicable to workloads of `type` 'CONTAINER'.
+`secretEnvironmentVariables.key` <br/>*string* | The location to obtain a value for a secret environment variable.
+`secretEnvironmentVariables.value` <br/>*string* | A secret environment variable's value.
+`deployments.instancesPerPop`<br/>*int* | The number of instances per point of presence. Only applicable if autoscaling is not enabled.
+`deployments.cpuUtilization` <br/>*int* | The percentage of CPU utilization. Only applicable if autoscaling is enabled.
+`deployments.minInstancesPerPop` <br/>*int* | The minimum number of instances per PoP. Only applicable if autoscaling is enabled. Should be greater than zero and less than 50.
+`deployments.maxInstancesPerPop` <br/>*int* | The maximum number of instances per PoP. Only applicable if autoscaling is enabled. Should be greater than zero and less than 50.
 
 <aside class="notice">
-A workload can be created without `publicPort`, `publicPortSrc`, `publicPortDesc` and `protocol` as part of the payload. But all these fields are required to open a port/port-range on the created workload.
+A workload can be created without any `ports`. However, `ports.publicPort` and `ports.protocol` are required to open a port or port range at workload creation.
```

Comment:
To be consistent with our translation standard, we should use add instead of create

Reply:
Fixed [here](https://github.com/cloudops/cloudmc-api-docs/pull/307/commits/dfd9768a21c2ec3faab74f6d17ec5881e7143f5f#diff-009b71b1c0eb5e3fe379e21d571a83b999e8661251e350ecb0efb9319d7a992eR419)